breeze

```diff
@@ -2853,10 +2853,14 @@ function breeze::check_and_save_all_params() {
     if [[ -n "${INSTALL_AIRFLOW_REFERENCE=}" ]]; then
         if [[ ${INSTALL_AIRFLOW_REFERENCE=} == *1_10* ]]; then
```

Comment:
In one of my latest  PRs (using constraints) i removed the leading '*' from those checks - this makes more sense for cases like version 2.1.10 :D

Reply:
Indeed :)